[Docker/Linux] test_disk_partitions failing inside Docker

If I run the tests suite inside a Docker container the `TestSystemAPIs.test_disk_partitions` fails.

```
======================================================================
FAIL: psutil.tests.test_system.TestSystemAPIs.test_disk_partitions
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/lib/python3.6/psutil/tests/test_system.py", line 508, in test_disk_partitions
    assert os.path.isdir(disk.mountpoint), disk
AssertionError: sdiskpart(device='/dev/sda5', mountpoint='/etc/resolv.conf', fstype='ext4', opts='rw,relatime,errors=remount-ro,data=ordered')
```
Of course `psutil` is woking fine and if I run in Python:

```python
import psutil as pu	
pu.disk_partitions(all=False)
[
sdiskpart(device='/dev/sda5', mountpoint='/data', fstype='ext4', opts='rw,relatime,errors=remount-ro,data=ordered'),
sdiskpart(device='/dev/sda5', mountpoint='/etc/resolv.conf', fstype='ext4', opts='rw,relatime,errors=remount-ro,data=ordered'),
sdiskpart(device='/dev/sda5', mountpoint='/etc/hostname', fstype='ext4', opts='rw,relatime,errors=remount-ro,data=ordered'),
sdiskpart(device='/dev/sda5', mountpoint='/etc/hosts', fstype='ext4', opts='rw,relatime,errors=remount-ro,data=ordered')
]
```
it correctly finds all the partitions.

The test is checking if the mount-point is a directory but that is not the case inside a Docker container where `/etc/resolv.conf` is mounted from the host.
https://github.com/giampaolo/psutil/blob/c10df5aa04e1ced58d19501fa42f08c1b909b83d/psutil/tests/test_system.py#L508

Is this something worth fixing?
Thank you for the project by the way :+1:

--- a/psutil/tests/test_system.py
+++ b/psutil/tests/test_system.py
@@ -501,11 +501,8 @@ class TestSystemAPIs(unittest.TestCase):
                 # we cannot make any assumption about this, see:
                 # http://goo.gl/p9c43
                 disk.device
-            if SUNOS or TRAVIS:
-                # on solaris apparently mount points can also be files
-                assert os.path.exists(disk.mountpoint), disk
-            else:
-                assert os.path.isdir(disk.mountpoint), disk
+            # on modern systems mount points can also be files
+            assert os.path.exists(disk.mountpoint), disk
             assert disk.fstype, disk
 
         # all = True

